modules/mysql/collect_queries_statistics.go

```diff
@@ -0,0 +1,52 @@
+package mysql
+
+import (
+	"time"
+)
+
+// Table Schema:
+// (MariaDB) https://mariadb.com/kb/en/information-schema-processlist-table/
+// (MySql) https://dev.mysql.com/doc/refman/5.7/en/information-schema-processlist-table.html
+const (
+	queryInfoSchemaProcessList = `SELECT TIME,USER FROM INFORMATION_SCHEMA.PROCESSLIST 
+WHERE Info IS NOT NULL AND Info NOT LIKE '%PROCESSLIST%' 
+ORDER BY TIME`
+)
+
+func (m *MySQL) collectProcessListStatistics(collected map[string]int64) error {
+	m.Debugf("executing query: '%s'", queryInfoSchemaProcessList)
+
+	var (
+		queryDuration              int64 // queryInfoSchemaProcessList execution time (not including row fetching)
+		longestRunningQuerySeconds int64 // slowest query milliseconds in process list
+		user                       string
+	)
+
+	start := time.Now()
+	rows, err := m.db.Query(queryInfoSchemaProcessList)
+	queryDuration = int64(time.Since(start).Seconds())
```

Comment:
let's make it milliseconds

Reply:
done, although this will make our unit tests flaky because on some slow machine the mocked duration will be 0 or 1 or a few more milliseconds but I'll figure something out

Reply:
I think we can just copy the value from `wantCollected`, like i did here https://github.com/netdata/go.d.plugin/blob/2181c601562b0d2fa9005756c3d141572f3b0758/modules/portcheck/portcheck_test.go#L64-L70

Reply:
Nice, I did something very similar